content/hardware/04.pro/boards/portenta-x8/tutorials/13.wordpress-webserver/content.md

```diff
@@ -64,39 +66,65 @@ services:
+
+Before installing the containers, make sure that no other container is running on the ports that the WordPress container will use. You can check what containers are running and what port they are using by running the `docker ps -a` command. This will show a list of the currently installed and running containers on the Portenta X8.
+
+To remove a container first stop it with `docker stop <container id>`, then you can run `docker rm <container id>` to remove it. If you want more information about handling containers on your Portenta X8, take a look at our [Managing Containers with Docker tutorial](https://docs.arduino.cc/tutorials/portenta-x8/docker-container).
 
-When you are in the correct directory and no other container is running on the ports that WordPress will use, you can now run `docker compose up -d`. Using the `-d` tag in the command will allow running these containers in the background. If you run the command without the `-d` tag, the application will exit when you close the terminal. When the command is executed it will start installing the **WordPress** and **MariaDB** containers. This can take a while. To get the output from the containers use: `docker-compose logs -f`. Once it is done you can connect to the device and site.
+When you are in the correct directory and no other container is running on the ports that WordPress will use, you can now run `docker compose up -d`. Using the `-d` tag in the command will allow running these containers in the background. If you run the command without the `-d` tag, the application will exit when you close the terminal.
+
+When the command is executed it will start installing the **WordPress** and **MariaDB** containers. This can take a while. To get the output from the containers use: `docker-compose logs -f`. Once it is done you can connect to the device and site.
 
 ![Containers install progress in the terminal](assets/webserver-container-install.png)
 
 ### Connecting to the WordPress Site
 
-To connect to the WordPress setup site, you simply need to access it with your Portenta X8s unique id and port. So for example: `http://portenta-x8-<uuid>.local:<port>`, where you would substitute the `<uuid>` with your Portenta X8's unique id and the port chosen for the WordPress container with `<port>`. The `<uuid>` can be found on the setup page that is shown in the [User Manual's Out-of-the-box experience](https://docs.arduino.cc/tutorials/portenta-x8/user-manual#out-of-the-box-experience), but you can also see it in the terminal when running `adb` or you can go to `http://192.168.7.1:8000` if you use Windows and Linux, on MacOS use `http://192.168.8.1:8000`.
+To connect to the WordPress setup site, you simply need to access it with your Portenta X8s unique id and port. So we can use the following address format:
+
+```
+http://portenta-x8-<uuid>.local:<port>
```

Comment:
Here, I think it could be nice to show the user that they can obtain the uuid by looking at the root id. For example, using the current tutorial's example image, the uuid is `3321ba09dab6fad9`. So the address is now "http://portenta-x8-3321ba09dab6fad9.local:8000". What do you think about this suggestion?

Reply:
I didn't know that! It is really interesting. Thanks!